site/guide/5-responses.md

```diff
@@ -59,7 +59,7 @@ use rocket::response::content;
 
 #[get("/")]
 fn json() -> content::Json<&'static str> {
-    content::Json("{ 'hi': 'world' }")
+    content::Json("{ \"hi\": \"world\" }")
```

Comment:
```rust
"{ \"hi\": \"world\" }"
// vs
r#"{ "hi": "world" }"#
```

The second is shorter (and IMO a bit cleaner), but do you think the `r#` syntax could confuse readers?

Reply:
I agree with the second one being a generally nicer solution, but I think it could be confusing for people new to rust or programming in general. Escaped characters in a string are a concept that's covered in most beginners guides for most languages. People who know about the second variation are probably not confused by the first one, I'm not sure whether that's the case the other way round.